src/Chat.tsx

```diff
@@ -86,7 +86,7 @@ export class Chat extends React.Component<ChatProps, {}> {
             this.store.dispatch<ChatActions>({ type: 'Set_Chat_Title', chatTitle });
         }
 
-        this.store.dispatch<ChatActions>({ type: 'Toggle_Upload_Button', showUploadButton: props.showUploadButton });
+        this.store.dispatch<ChatActions>({ type: 'Toggle_Upload_Button', showUploadButton: props.showUploadButton !== false });
```

Comment:
so, if `showUploadButton` is `undefined`, you want it to be shown?

Reply:
Yes, the default behavior should show upload button.